docs/docs/providers/openai_responses_limitations.mdx

```diff
@@ -5,52 +5,34 @@ sidebar_label: Limitations of Responses API
 sidebar_position: 1
 ---
 
-## Unresolved Issues
+## Issues
 
 This document outlines known limitations and inconsistencies between Llama Stack's Responses API and OpenAI's Responses API. This comparison is based on OpenAI's API and reflects a comparison with the OpenAI APIs as of October 6, 2025 (OpenAI's client version `openai==1.107`).
 See the OpenAI [changelog](https://platform.openai.com/docs/changelog) for details of any new functionality that has been added since that date. Links to issues are included so readers can read about status, post comments, and/or subscribe for updates relating to any limitations that are of specific interest to them. We would also love any other feedback on any use-cases you try that do not work to help prioritize the pieces left to implement.
 Please open new issues in the [meta-llama/llama-stack](https://github.com/meta-llama/llama-stack) GitHub repository with details of anything that does not work that does not already have an open issue.
 
-### Instructions
-**Status:** Partial Implementation + Work in Progress
-
-**Issue:** [#3566](https://github.com/llamastack/llama-stack/issues/3566)
-
-In Llama Stack, the instructions parameter is already implemented for creating a response, but it is not yet included in the output response object.
-
----
-
-### Streaming
+### Web-search tool compatibility
 
 **Status:** Partial Implementation
 
-**Issue:** [#2364](https://github.com/llamastack/llama-stack/issues/2364)
-
-Streaming functionality for the Responses API is partially implemented and does work to some extent, but some streaming response objects that would be needed for full compatibility are still missing.
-
----
-
-### Prompt Templates
+Both OpenAI and Llama Stack support a web-search built-in tool.  The [OpenAI documentation](https://platform.openai.com/docs/api-reference/responses/create) for web search tool in a Responses tool list says:
 
-**Status:** Partial Implementation
+> The type of the web search tool. One of `web_search` or `web_search_2025_08_26`.
 
-**Issue:** [#3321](https://github.com/llamastack/llama-stack/issues/3321)
+Llama Stack now supports both `web_search` and `web_search_2025_08_26` types, matching OpenAI's API. For backward compatibility, Llama Stack also supports `web_search_preview` and `web_search_preview_2025_03_11` types.
```

Comment:
This seems confusing now.  I think it would make sense to split this into the parts that are still open (filters and user location) that stay here, and the parts that are now done (the aliases) in the Resolved Issues section, which I think requires some rewriting of both.